using-virtual-machines/guest-agent.adoc

```diff
@@ -0,0 +1,161 @@
+Guest Agent information from the Virtual Machine
+------------------------------------------------
+
+Guest Agent (GA) is an optional component running inside of the Virtual Machine.
```

Comment:
Guest Agent (GA) is an optional component that can run inside of Virtual Machines.